Packet++/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ add_library(
   src/SingleCommandTextProtocol.cpp
   src/SipLayer.cpp
   src/SllLayer.cpp
+  src/Sll2Layer.cpp
```

Comment:
For some reason `cmake-format` fails and claims that `src/SllLayer.cpp` should come after `src/Sll2Layer.cpp`...

https://github.com/seladb/PcapPlusPlus/actions/runs/5617676562/job/15222130805?pr=1138

Reply:
done